tests/data/dataset_readers/nlvr_test.py

```diff
@@ -5,6 +5,7 @@
 
 
 class TestNlvrDatasetReader(AllenNlpTestCase):
+    # TODO(pradeep): Write a test for reading grouped and processed data.
```

Comment:
I forgot what that comment refers to. It's probably out of date, because the test for processed data works with both grouped and ungrouped files. Removed it.